collectors/proc.plugin/proc_net_dev.c

```diff
@@ -5,6 +5,19 @@
 #define PLUGIN_PROC_MODULE_NETDEV_NAME "/proc/net/dev"
 #define CONFIG_SECTION_PLUGIN_PROC_NETDEV "plugin:" PLUGIN_PROC_CONFIG_NAME ":" PLUGIN_PROC_MODULE_NETDEV_NAME
 
+// As defined in https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-class-net
+const char *operstate_names[] = { "unknown", "notpresent", "down", "lowerlayerdown", "testing", "dormant", "up" };
+
+static inline int get_operstate(char *operstate)
+{
+    for (int i = 0; i < (int) (sizeof(operstate_names) / sizeof(char *)); i++) {
```

Comment:
This line appears to be causing issues in Travis: https://travis-ci.com/netdata/netdata/jobs/262680560#L4684

Based on the error message, the only change that should be needed is moving the declaration of `i` out of the for loop.

Reply:
@stelfrag another possibility would be to add -std=c99 when we are compiling collectors, I remember that we decided to use c99 in our codes, but it looks like that we did not adjust the compiler options for this.

Reply:
@vlvkobal ^^ 😄

Reply:
C99 support was already added to the master branch a few days ago.

Reply:
A rebase seems to be in order then prior to merging. Given the lack of conflicts, it should just be a trivial one too.

Reply:
A merge of the master branch will be enough.